DeAceroDLabs/marketplace-frontend
# Feature/dmp 434

## What is done

[[DMP-434] Popup component](https://deacero.atlassian.net/browse/DMP-434)

## How is done

- Updated the `Popup` component with new props.

## How to test

1. Run the project via `npm start`
2. You should see the Home Screen.
3. Click on contact to see popup
